src/common.build.pre.props

```diff
@@ -110,8 +109,12 @@
         C26813: Use 'bitwise and' to check if a flag is set.
           The MSVC v19.31 toolset has a bug where a pointer to an enum is incorrectly flagged with C26813.
           It's supposed to be fixed with VS 17.2.1 and 17.3.0 and later respectively.
+        C26493: Don't use C-style casts (type.4).
+          The MSVC 14.33.31629 toolset (VS 17.3 Preview 5) has a bug
+          where brace initializations are treated as C-style casts.
       -->
-      <DisableSpecificWarnings>4201;4312;4467;5105;26445;26813;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings Condition="'$(VCToolsVersion)' != '14.33.31629'">4201;4312;4467;5105;26445;26813;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings Condition="'$(VCToolsVersion)' == '14.33.31629'">4201;4312;4467;5105;26445;26813;26493;%(DisableSpecificWarnings)</DisableSpecificWarnings>

```

Comment:
Is this gonna turn into a `'$(VCToolsVersion)' &gt; '14.33.31629' && '$(VCToolsVersion)' &lt; 'some future version'` in the future?

Reply:
I'm not sure why I never noticed but apparently this has been an issue for longer than that: https://godbolt.org/z/oanYPnzoc
So I'll just silently repurpose this PR to only remove the few `Condition` checks we don't need.